packages/expo-av/CHANGELOG.md

```diff
@@ -38,6 +38,10 @@
 
 - Add methods to get and set audio recording inputs. ([#15806](https://github.com/expo/expo/pull/15806) by [@computerjazz](https://github.com/computerjazz))
 
+
+### 🐛 Bug fixes
+- Fix expo-av ExoPlayer cannot adapt the video quality to user bandwidth fast enough on Android. ([#16455](https://github.com/expo/expo/pull/16455) by [@artiya4u](https://github.com/artiya4u))
+
```

Comment:
This entry should be placed far closer to the top of this file, under `## Unpublished` section 👀 
```suggestion
```